polygon-id/id3/id3.json

```diff
@@ -1,10 +1,10 @@
 {
   "$metadata": {
     "uris": {
-      "jsonLdContext": "https://raw.githubusercontent.com/NexeraProtocol/vc-schemas/development/polygon-id/id3/id3.jsonld"
+      "jsonLdContext": "https://raw.githubusercontent.com/NexeraProtocol/vc-schemas/fix-id3-country-type/polygon-id/id3/id3.jsonld"
```

Comment:
i don't think we should be updating this, we should use the github action to trigger a deploy of vc-schemas from a branch.. it is good for now, but don't forget to revert back before merging

Reply:
this should be reverted at this point.